components/dashboard/src/settings/settings-menu.ts

```diff
@@ -36,7 +36,7 @@ export default function getSettingsMenu(params: {
         ...(params.enablePersonalAccessTokens
             ? [
                   {
-                      title: "Personal Access Tokens",
+                      title: "Access Tokens",
```

Comment:
**question**: What do you think of moving this further down the menu just above _Preferences_ as this is not a menu that will be used often by users?

| BEFORE | AFTER |
|-|-|
| <img width="340" alt="sidebar-before" src="https://user-images.githubusercontent.com/120486/203512776-f5268d61-52a7-45f5-b048-e11fb8e36154.png"> | <img width="340" alt="sidebar-after" src="https://user-images.githubusercontent.com/120486/203512793-d1006544-5a48-4f07-b4ac-20007d2777c0.png"> |